### PR TITLE
Match func_8001C524

### PR DESCRIPTION
**Pull request by tonyspumoni**
thanks to @Ryan-Myers & @DavidSM64 for help on `func_80072348`

### CODE REVIEW

File: src/objects.c [hunk 1]
```diff
@@ -1604,7 +1605,51 @@ s32 func_8001C48C(s32 arg0) {
...
+    f32 x;
+    f32 len;
+    f32 z;
+    f32 y;
+    f32 dist;
+    s32 var_a0;
+    s32 numSteps;
+    s32 result;
+    ObjectSegment* segment;
+    LevelObjectEntry_TTDoor* levelObj;
```
Comment (DavidSM64) [inline src/objects.c:1620]:
```suggestion
    ObjectSegment *segment;
    LevelObjectEntry_TTDoor *levelObj;
```

**Review by DavidSM64 — COMMENTED** [review]
Unfortunately, decomp.me likes to set the asterisks to the left with no way to change it.

File: src/objects.c (hunk 1 above)
Comment (tonyspumoni) [reply src/objects.c:1620]: ah, ill make a note to try and remember to swap these, thanks for the fix

**Review by DavidSM64 — APPROVED** [review]
LGTM

**Pull request merged**